### PR TITLE
[AR] Translation of 06-2.md

### PR DESCRIPTION
**Pull request by AliEl-01**
translation of 06-2.md , and also creating a folder call week06 to manage and organize all week06 files

### CODE REVIEW

File: docs/ar/week06/06-2.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
+lang-ref: ch.06-2
+lecturer: Yann LeCun
+title: RNNs, GRUs, LSTMs, Attention, Seq2Seq, and Memory Networks
+authors: Jiayao Liu, Jialing Xu, Zhengyang Bian, Christina Dominguez
+date: 2 March 2020
```
Comment (Atcold) [inline docs/ar/week06/06-2.md:5]: What happened here? There is no header, the title is not translated, missing many fields.
Please, update.

File: docs/ar/week06/06.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+lang-ref: ch.06
```
Comment (Atcold) [inline docs/ar/week06/06.md:2]: `lang: ar` is missing.

File: docs/_config.yml [hunk 3]
```diff
@@ -425,6 +425,10 @@ ar:
     - path: ar/week04/04.md
       sections:
         - path: ar/week04/04-1.md
+  chapters:
+    - path: ar/week06/06.md
+      sections:
+        - path: ar/week06/06-1.md
```
Comment (Atcold) [inline docs/_config.yml:431]: `06-2` is missing.

**Review by Atcold — CHANGES_REQUESTED** [review]
A few issues. See below.

**Comment by AliEl-01** [comment]
is everythings 0k now !!!

**Comment by Atcold** [comment]
Why did you close this PR? I'm confused.

**Comment by AliEl-01** [comment]
I did another one with some updated stuff!😕

**Comment by Atcold** [comment]
A PR keeps been updated if you push new commits.
You don't need to create a new one.

**Pull request closed without merging**